server/services/reportedAdjudicationsService.ts

```diff
@@ -36,6 +37,70 @@ export default class ReportedAdjudicationsService {
     return new ManageAdjudicationsClient(user.token).getReportedAdjudication(adjudicationNumber)
   }
 
+  async getReviewDetails(adjudicationNumber: number, user: User) {
+    const adjudicationData = await new ManageAdjudicationsClient(user.token).getReportedAdjudication(adjudicationNumber)
+    const { reportedAdjudication } = adjudicationData
+
+    if (reportedAdjudication.status === ReportedAdjudicationStatus.AWAITING_REVIEW)
+      return { reviewStatus: reportedAdjudicationStatusDisplayName(reportedAdjudication.status) }
+
+    const reviewingOfficer =
+      reportedAdjudication.reviewedByUserId &&
+      (await this.hmppsAuthClient.getUserFromUsername(reportedAdjudication.reviewedByUserId, user.token))
+
+    const getReasonTitle = (status: ReportedAdjudicationStatus) => {
```

Comment:
All the code below this ought not to be in a service as it's completely UI-related. Probably the best place for all this UI-related code is the njk file (or the nunjucksSetup maybe). 

But it's not worth it right now as i suspect there is a lot of UI-specific code already in the services.